plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_launcher_util.py

```diff
@@ -112,7 +110,10 @@ def ray_tmp_dir(yaml_path: str, docker: bool) -> Generator:
     _run_command(rmtemp_args)
 
 
-def ray_working_dir(yaml_path: str, docker: bool) -> str:
+def ray_home_dir(yaml_path: str, docker: bool) -> str:
```

Comment:
This function does not seem to return a homedir, instead it looks like it's returning a pem file.

Reply:
was this comment meant for another function?